### DISCUSSION TITLE
Phone number

### DISCUSSION THREAD

**Discussion by Djentelman57** [OFFLINE-GmbH/oc-mall-plugin#1066] [Q&A]
Help me with adding a phone number to your profile and placing an order

**Comment by ismaeil-1** [comment]
Have you found a solution?

**Comment by willouch** [comment]
UserPlus plugin ?
https://octobercms.com/plugin/rainlab-userplus

**Reply by ismaeil-1** [reply]
No issues here man I learned how to extend plugins
